sdk/examples/devmode_rust/src/engine.rs

```diff
@@ -253,6 +281,35 @@ impl Engine for DevmodeEngine {
                             service.initialize_block();
                         }
 
+                        Update::PeerMessage(message, sender_id) => match message
+                            .message_type
+                            .as_ref()
+                        {
+                            "published" => {
```

Comment:
Any reason why these are strings as opposed to enums?

Reply:
Added `DevmodeMessage` enum